storage/google/cloud/storage/bucket.py

```diff
@@ -770,6 +770,11 @@ def list_blobs(
         :returns: Iterator of all :class:`~google.cloud.storage.blob.Blob`
                   in this bucket matching the arguments.
         """
+        warnings.warn(
+            "Bucket.list_blobs method is deprecated. Use Client.list_blobs instead.",
+            PendingDeprecationWarning,
+            stacklevel=2,
+        )
```

Comment:
Not sure about this. It will warn if you call Client.list_blobs too

Reply:
Let's wait to mark these as deprecated until we move all the actual implementation out of the Blob and Bucket classes into the Client class.